src/compiler/parser.ts

```diff
@@ -2143,16 +2166,31 @@ module ts {
             }
         }
 
+        function setContextParameters(explicitFlags: ParserContextFlags = ParserContextFlags.None, ambientFlags: ParserContextFlags = ParserContextFlags.None) {

```

Comment:
1) There are a few calls but I can inline the values.
2) I could make them optional, but I think changing the call-sites to explicitly pass the value is cleaner.